.github/workflows/ant.yaml

```diff
@@ -174,3 +174,11 @@ jobs:
           fields: repo,message,commit,author,action,eventName,ref,workflow,job
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  propose:
+    needs: [publish]
+    if: needs.publish.result == 'success'
+    uses: ./.github/workflows/vaot-proposal.yaml
+    secrets: inherit
+    with:
+      published_module_id: ${{ needs.publish.outputs.published_module_id }}
```

Comment:
normally, i would start with just a manual workflow dispatch and test once or twice before automating all the way through. it may be beneficial for your dry-run script to have a dry-run flag that allows for running without actually posting anything to the vaot process